cockatrice/src/tab_game.h

```diff
@@ -117,7 +117,7 @@ class TabGame : public Tab
 {
     Q_OBJECT
 private:
-    QTimer *gameTimer;
+    QTimer *gameTimer{};
```

Comment:
why add default initializers to all these?

Reply:
Clangify did that, as it said some variables were missed. I'll revert this